gloo/types.h

```diff
@@ -104,6 +113,13 @@ struct __attribute__((__aligned__(2))) float16 {
     x = res.x;
   }
 
+#ifdef _WIN64
```

Comment:
Make the `unsigned long` above into `unsigned long long` to match semantics with Linux.